roles/pulp3-content/README.md

```diff
@@ -0,0 +1,16 @@
+pulp3-content
+=============
+
+Install, configure, and set the state of pulp content app.
+
```

Comment:
You should also document the `pulp_content_all` variable even though it has a default.

Reply:
Instead of using and documenting the variable, I removed it